assignment/impl/src/java/org/sakaiproject/assignment/impl/persistence/AssignmentRepositoryImpl.java

```diff
@@ -139,6 +162,16 @@ public AssignmentSubmission findSubmission(String submissionId) {
         return (AssignmentSubmission) sessionFactory.getCurrentSession().get(AssignmentSubmission.class, submissionId);
     }
 
+    @Override
+    public AssignmentSubmissionSubmitter findSubmissionSubmitter(String submitterId) {
```

Comment:
I don't understand why you sould take a submitterId (aka a userId) turn it into a long and query the table?

Reply:
findSubmissionSubmitter and getSubmissionSubmitter aren't used anymore.  Both deleted.